pkg/volume/cinder/cinder_util.go

```diff
@@ -132,6 +132,40 @@ func (util *CinderDiskUtil) DetachDisk(cd *cinderVolumeCleaner) error {
 	return nil
 }
 
+func (util *CinderDiskUtil) DeleteVolume(cd *cinderVolumeDeleter) error {
+	cloud := cd.plugin.host.GetCloudProvider()
```

Comment:
Create a common util function `GetCiderCloudProvider(...)` to avoid code duplication.